data/transactions/logic/assembler.go

```diff
@@ -1567,41 +1591,62 @@ func (ops *OpStream) assemble(text string) error {
 		if ops.versionedPseudoOps == nil {
 			ops.versionedPseudoOps = prepareVersionedPseudoTable(ops.Version)
 		}
-		opstring := fields[0]
-		if opstring[len(opstring)-1] == ':' {
-			ops.createLabel(opstring[:len(opstring)-1])
-			fields = fields[1:]
-			if len(fields) == 0 {
-				ops.trace("%3d: label only\n", ops.sourceLine)
+		var current []string
+		var next []string
+		next = fields
+		for current, next = processFields(next); len(current) > 0 || len(next) > 0; current, next = processFields(next) {
```

Comment:
I think this is right:
```suggestion
		for current, next := processFields(fields); len(current) > 0 || len(next) > 0; current, next = processFields(next) {
```